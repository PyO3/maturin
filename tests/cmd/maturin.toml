bin.name = "maturin"
args = "--help"
