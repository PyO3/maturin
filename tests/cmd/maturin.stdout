Build and publish crates with pyo3, cffi and uniffi bindings as well as rust binaries as python
packages

Usage: maturin[EXE] [OPTIONS] <COMMAND>

Commands:
  build        Build the crate into python packages
  publish      Build and publish the crate as python packages to pypi
  list-python  Search and list the available python installations
  develop      Install the crate as module in the current virtualenv
  sdist        Build only a source distribution (sdist) without compiling
  init         Create a new cargo project in an existing directory
  new          Create a new cargo project
  generate-ci  Generate CI configuration
  upload       Upload python packages to pypi
  help         Print this message or the help of the given subcommand(s)

Options:
  -v, --verbose...
          Use verbose output.
          
          * Default: Show build information and `cargo build` output. * `-v`: Use `cargo build -v`.
          * `-vv`: Show debug logging and use `cargo build -vv`. * `-vvv`: Show trace logging.
          
          You can configure fine-grained logging using the `RUST_LOG` environment variable.
          (<https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives>)

  -h, --help
          Print help (see a summary with '-h')

  -V, --version
          Print version

Visit https://maturin.rs to learn more about maturin.
